nw-firestore-schema.ts

```diff
@@ -105,3 +105,8 @@ interface Root {
   Hackathons: Hackathon[];
   FAQ: FAQ[];
 }
+
+interface Livesite {
+  imgURL: string; // url to logo image
```

Comment:
should this be hackathonImgUrl instead to make it more clear

Reply:
Maybe, technically its possible it is not be a hackathon image, ie a designer could make it say "LIVESITE"?